### ISSUE TITLE
AI has lost non-widescreen support

### ISSUE THREAD

**Issue by Otterwatch** [tgstation/TerraGov-Marine-Corps#13851]
### Description

I use non-widescreen with AI and see my entire view range scrunched to the left half of the screen.

### Test Merges

[#13515: 'Adds 2 new tadpole models updating: to make grief poles less griefy' by XANAv2 at commit e52d05bff0](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13515)
[#13626: 'Slumbridge' by chizzy376 at commit da12c5ca11](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13626)
[#13191: 'New Tier 3 caste: Behemoth' by lKiseki at commit 030d18bf30](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13191)
[#13825: 'Ports tg's del the world unit test' by ivanmixo at commit 17d6f4a738](https://github.com/tgstation/TerraGov-Marine-Corps//pull/13825)

### Reproduction Steps

1. Turn off Widescreen.
2. Join as AI.
3. Look.

### Screenshots

![I'm so sad](https://github.com/tgstation/TerraGov-Marine-Corps/assets/77871802/3481bb1a-6eb8-4b4a-bc86-d10baee3ab74)

**Comment by lbnesquik** [comment]
How did it look before?

**Comment by Otterwatch** [comment]
> How did it look before?

The buttons at the bottom left were at the bottom left of the game panel which has been replaced with complete darkness (black). Everything in that image that is black is the game window not displaying to its borders. It should look the same as widescreen but with two less tiles of vision across and a smaller game window.

**Comment by lbnesquik** [comment]
Did you end up managing to reproduce this after I went to bed?

**Comment by Otterwatch** [comment]
Yes, I sent you a dm about it. If anyone else would like to look at it, I can send the file to you if you dm me. Otto Headmin. The file is sadly too big for git.

**Comment by Otterwatch** [comment]
https://cdn.discordapp.com/attachments/1145076601596424202/1145076635415089242/AI.mp4

**Comment by lbnesquik** [comment]
!(image)[https://cdn.discordapp.com/attachments/1145076601596424202/1145076635415089242/AI.mp4]

**Comment by Otterwatch** [comment]
I have tried reverting and compiling up to 3 months back now and the issue persists. The only help I can provide if anyone looks into this in the future is that the last time I noticed non widescreen worked was [August 12th](https://github.com/tgstation/TerraGov-Marine-Corps/issues/13769). The next AI related PR was [RipGrayson's pr to give AI the ability to bioscan](https://github.com/tgstation/TerraGov-Marine-Corps/pull/13526) and also Multicamera + Add camera are now on the same y axis as the rest of the bottom row instead of being slightly above them. I have no idea what could've caused this.
